universal-application-tool-0.0.1/tags.txt

```diff
@@ -0,0 +1,117 @@
+a
```

Comment:
This is a list of all the html tags. I had to add them for some reason to make it work

Reply:
I think I just realized why I had to do this. I guess tailwind parses which html tags are being used when creating its .css, so I need to parse which `j2html` tags are being used. This would probably bring down the size a bit from where it is now (155K)